PocketSwift/Packages/Core/Model/Relay.swift

```diff
@@ -16,35 +16,54 @@ import Foundation
  - devId: The id used to interact with Pocket Api.
  - network: The blockchain network name, ie: ETH, AION.
  - netID: The netid of the Blockchain.
- 
+ - httpMethod: HTTP Method for rest api requests, GET, POST, DELETE, etc.
+ - path: URL path for the rest api request.
+ - queryParams: 
  */
 public class Relay: Model, Input {
     let network: String
     let netID: String
     let data: String
     let devID: String
+    let httpMethod: String;
+    let path: String;
     
-    init(network: String, netID: String, data: String, devID: String) {
+    init(network: String, netID: String, data: String?, devID: String, httpMethod: String?, path: String?, queryParams: [String: String]?) {
```

Comment:
Why not make a ENUM with POST, PUT, DELETE, HEAD and GET instead of a string for the `httpMethod`

Reply:
true that